ci/scripts/jenkins/build.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source ${WORKSPACE}/ci/scripts/jenkins/common.sh
+
+rm -rf ${SRF_ROOT}/.cache/ ${SRF_ROOT}/build/
+
+gpuci_logger "Creating conda env"
+mamba env create -n srf -q --file ${CONDA_ENV_YML}
+conda deactivate
+conda activate srf
+
+mamba env update -q -n srf --file ${SRF_ROOT}/ci/conda/environments/ci_env.yml
+
+CMAKE_CACHE_FLAGS="-DCCACHE_PROGRAM_PATH=$(which sccache) -DSRF_USE_CCACHE=ON"
+
+gpuci_logger "Check versions"
+python3 --version
+cmake --version
+ninja --version
+
+if [[ "${BUILD_CC}" == "gcc" ]]; then
+    gpuci_logger "Building with GCC"
+    gcc --version
+    g++ --version
+    CMAKE_FLAGS="${CMAKE_BUILD_ALL_FEATURES} ${CMAKE_CACHE_FLAGS} -DSRF_USE_IWYU=ON"
```

Comment:
Why is IWYU enabled during the build here? I've always had issues with running tools like IWYU and clang-tidy with ccache. Isnt IWYU going to be checked in `cpp_checks.sh`?

Reply:
Oh, good point.
I think this was just a copy/paste from the previous CI pipeline.